docs/advanced-topics/snapshot-and-doi.rst

```diff
@@ -0,0 +1,101 @@
+Creating Snapshots and Requesting DOIs
+=============================================
+
+Introduction
+------------------
+Starting in Dockstore 1.7.0, users can create snapshots and Digital
+Object Identifiers (DOIs) for their workflows. The ‘Snapshot’ and
+‘Request DOI’ actions can be found on the versions tab of any published
+workflow on the ‘My Workflows’ page. These features are specific to
+individual versions of a workflow entry.
+
+.. image:: ../assets/images/docs/snapshot_doi/snapshot_doi_versiontab.png
+
+
+
+Connect Zenodo Account
+----------------------
+Dockstore uses `Zenodo <http://about.zenodo.org/>`__ as its DOI provider. A Zenodo account is
+required to request a DOI for a workflow version. Link your Zenodo credentials to your Dockstore account on the
+`accounts page <https://dockstore.org/accounts?tab=accounts>`__. Requesting a DOI on Dockstore will
+create a public entry on Zenodo and upload associated files.
+
+.. image:: ../assets/images/docs/snapshot_doi/link_zenodo.png
+
+
+
+Create Snapshot
+----------------
+A snapshot is a point-in-time capture of the descriptor(s), test parameter file(s),
+and metadata associated with a workflow version. Snapshotting a version will also
```

Comment:
Depending on what you mean by metadata, this is not necessarily true. The Dockstore metadata, e.g., platform verification can be modified.

Reply:
True, can look into `io.dockstore.webservice.core.VersionMetadata` to indicate what can change